dev-tools/mage/crossbuild.go

```diff
@@ -134,16 +134,37 @@ func CrossBuild(options ...CrossBuildOption) error {
 		opt(&params)
 	}
 
-	// Docker is required for this target.
-	if err := HaveDocker(); err != nil {
-		return err
-	}
-
 	if len(params.Platforms) == 0 {
 		log.Printf("Skipping cross-build of target=%v because platforms list is empty.", params.Target)
 		return nil
 	}
 
+	// AIX can't really be crossbuilt, due to cgo and various compiler shortcomings.
+	// If we have a singular AIX platform set, revert to a native build toolchain
+	if runtime.GOOS == "aix" {
+		for _, platform := range params.Platforms {
```

Comment:
nit: `if len(params.Platforms) == 1 and params.Platforms[0].GOOS() == "aix"` and you save a couple of indentation levels